rector.php

```diff
@@ -8,5 +8,25 @@
     $parameters = $containerConfigurator->parameters();
     $parameters->set(Option::SETS, [
         DowngradeSetList::PHP_80,
+
+        //PHP73 DowngradeSetList::PHP_74,
+
+        //PHP72 DowngradeSetList::PHP_74,
+        //PHP72 DowngradeSetList::PHP_73,
+
+        //PHP71 DowngradeSetList::PHP_74,
+        //PHP71 DowngradeSetList::PHP_73,
+        //PHP71 DowngradeSetList::PHP_72,
+
+        //PHP70 DowngradeSetList::PHP_74,
+        //PHP70 DowngradeSetList::PHP_73,
+        //PHP70 DowngradeSetList::PHP_72,
+        //PHP70 DowngradeSetList::PHP_71,
+
+        //PHP56 DowngradeSetList::PHP_74,
+        //PHP56 DowngradeSetList::PHP_73,
+        //PHP56 DowngradeSetList::PHP_72,
+        //PHP56 DowngradeSetList::PHP_71,
+        //PHP56 DowngradeSetList::PHP_70,
```

Comment:
I like this solution! For Rector, I had implemented many config files for each case (`rector-php74.php`, `rector-php73.php`, etc), which is very bureaucratic. This is much better!